app/forms/thredded/topic_form.rb

```diff
@@ -36,6 +36,7 @@ def save
       ActiveRecord::Base.transaction do
         topic.save!
         post.save!
+        UserTopicReadState.read_on_first_post!(user, topic) if topic.previous_changes.include?(:id)
```

Comment:
@glebm I think we ought to have a feature spec that checks that the private topic is read for the creator. I don't think it matters how it happens, but the feature spec should then catch the change. I started creating the test, but it was quite a high bar as there is no existing similar feature spec, and I find the style of these feature specs quite heavyweight and hard-work to create. I could create a quick and light one which is just a capybara spec, if you like?